assignments/models.py

```diff
@@ -59,6 +67,70 @@ class Meta:
         verbose_name = _('open text task')
         verbose_name_plural = _('open text tasks')
 
+
+class BudgetTextTask(models.Model):
```

Comment:
Why is it called `BudgetTextTask` and not just `BudgetTask` or better yet `BudgetingTask`?